e2e-tests/wait_for_streaming.sh

```diff
@@ -47,7 +47,7 @@ function start_pipeline() {
     cd /workspace/pipelines
     ../utils/start_pipelines.sh -s -streamingLog /workspace/e2e-tests/log.log \
     -u ${OPENMRS_URL}/openmrs  -o /workspace/e2e-tests/STREAMING \
-    -secondsToFlushStreaming 15 -fhirSinkPath ${SINK_SERVER}/fhir \
+    -secondsToFlushStreaming 5 -fhirSinkPath ${SINK_SERVER}/fhir \
```

Comment:
is there a reason we reduce this?

Reply:
Ah, is it due to the race condition below?

Reply:
Correct; that's not a robust fix but just reduces the likelihood (beside, the root cause of the test failure was not really this; at the time I was tweaking these, I did not know the MDM issue yet).